src/app/qgisapp.cpp

```diff
@@ -806,6 +807,7 @@ QgisApp::QgisApp( QSplashScreen *splash, bool restorePlugins, bool skipVersionCh
   // Advanced Digitizing dock
   startProfile( QStringLiteral( "Advanced digitize panel" ) );
   mAdvancedDigitizingDockWidget = new QgsAdvancedDigitizingDockWidget( mMapCanvas, this );
+  mAdvancedDigitizingDockWidget->toggleViewAction()->setShortcut( tr( "Alt+4", "Keyboard shortcut: Show advanced digitize panel." ) );
```

Comment:
I don't know how important this is nor whether it's at all but right naming of the panel, if I'm not wrong, is "advancrd digitizing"